robot/client/client.go

```diff
@@ -363,6 +372,31 @@ func New(ctx context.Context, address string, clientLogger logging.ZapCompatible
 		}
 	}
 
+	// if withNetworkStats is set, there are latencyPingNum gRPC requests sent to the server
+	// to measure the average latency of the connection, which is then logged to the user.
+	if rOpts.withNetworkStats {
+		totalTime := 0.0
+		for range latencyPingNum {
+			startTime := time.Now()
+			_, err := rc.client.GetOperations(ctx, &pb.GetOperationsRequest{})
```

Comment:
Could you leave a short comment above this line saying we use `GetOperations` as that is what app.viam.com's "Control" tab uses to get latency?